qiskit/circuit/quantumcircuit.py

```diff
@@ -1165,6 +1166,20 @@ def append(
             if is_parameter:
                 instruction = copy.deepcopy(instruction)
 
+        # Deprecate the passing multiple qargs for contolled gates
+        if len(qargs) == 2:  # check if singly-controlled gate
+            # check for multiple qubits
+            if any(isinstance(qarg, list) and len(qarg) > 1 for qarg in qargs):
+                warnings.warn(
+                    "Creating multiple single-controlled gates with a single call, "
+                    "such as QuantumCircuit.cx([0, 1], 2) is deprecated as of Qiskit "
+                    "Terra 0.19.0 and will be removed no sooner than 3 months after it's "
```

Comment:
Will the new behavior be introduced at the same time that the existing one is removed? If that's the case, it is worth noting that existing calls might still work syntactically, but will have different behavior if not migrated?

Reply:
Well if we can change it directly that'd be nice since we need one less release cycle. Maybe @mtreinish can weigh in here, but currently (with 0eb0a6574) I've changed it so that we'll change it directly.